.github/workflows/lint-backend.yml

```diff
@@ -1,6 +1,5 @@
 name: Lint Backend
-on: 
-  pull_request:
+on: [push, pull_request]
```

Comment:
The push will trigger for pushes to all branches. e.g. this branch ran the checks twice, once for the push and the once pr.
Do we want to target the push just to `main` (and add any other protected branches later once branch naming confirmed) ?

Reply:
Yeah wasn't sure about this, probably some benefits to having it but maybe not worth the unsightly 8 checks in PRs.